components/addContact/index.test.jsx

```diff
@@ -80,14 +80,20 @@ describe("handleSubmit", () => {
   let alertError;
   let alertSuccess;
   let setDirtyForm;
-  const fetch = "fetch";
+  let session;
 
   beforeEach(() => {
     setAgentId = jest.fn();
     setIsLoading = jest.fn();
     alertError = jest.fn();
     alertSuccess = jest.fn();
     setDirtyForm = jest.fn();
+    session = {
```

Comment:
Could've used `mockSession` for this as well. (But not necessary, so I'll let you decide what you want to do.)